contracts/ERC20StakingRewardsDistribution.sol

```diff
@@ -186,16 +228,48 @@ contract ERC20StakingRewardsDistribution {
             uint256 _recoverableRewards =
                 IERC20(_reward.token).balanceOf(address(this)) -
                     (_reward.amount - _reward.claimed);
-            if (!_atLeastOneNonZeroRecovery && _recoverableRewards > 0)
-                _atLeastOneNonZeroRecovery = true;
-            _recoveredUnassignedRewards[_i] = _recoverableRewards;
-            IERC20(_reward.token).safeTransfer(owner, _recoverableRewards);
+            if (_recoverableRewards > 0) {
+                _recoveredUnassignedRewards[_i] = _recoverableRewards;
+                IERC20(_reward.token).safeTransfer(owner, _recoverableRewards);
+            }
         }
-        require(_atLeastOneNonZeroRecovery, "SRD22");
```

Comment:
Could also remove the error code from the top list, unless it's annoying with renumbering...

Reply:
Solved in a610a9c663acfcbcf36d9b0dde5136d2ade18599